android/app/src/main/java/org/electroncash/electroncash3/Main.kt

```diff
@@ -403,6 +411,7 @@ class WalletOpenDialog : PasswordDialog<String>() {
         super.onBuildDialog(builder)
         builder.setView(R.layout.wallet_open)
                 .setNeutralButton(R.string.Delete_wallet, null)
```

Comment:
On the emulator with API level 28, this dialog box shows up with some extra space below the buttons:

<img src="https://user-images.githubusercontent.com/166954/71092919-104a2500-21a0-11ea-96cf-a13661015136.png" width="300"/>

From experiment, it looks like this is caused by the length of the "delete wallet" message. Looks like a known bug in the dialog library (https://github.com/material-components/material-components-android/issues/742). Unless you can work out how to fix this, let's just reduce the message to "delete".

Reply:
I had a dig through the depths of this to try to find some trick to avoid the problem without success.  So, I've just changed the button label to "delete" as you suggest.

However, I'm concerned that someone might misinterpret the delete button as meaning "delete the text I have just entered".  The original confirmation message doesn't help dispel this misinterpretation either as it simply says: `Do you want to delete 'foo'?` (had you just typed 'foo').  So I've strengthened the confirmation message a bit to instead read `Are you sure you want to delete your wallet 'foo'?`.